scripts/qa_crowbarsetup.sh

```diff
@@ -2996,6 +2996,15 @@ function get_first_node_from_cluster
                         ['elements']['pacemaker-cluster-member'].first"
 }
 
+function get_all_nodes_from_cluster
+{
+    local cluster=$1
+    crowbar pacemaker proposal show $cluster | \
```

Comment:
I think it would be better if we switched to `crowbarctl`. For instance:
```
crowbarctl proposal show pacemaker $cluster --json
```

Reply:
fixed